src/Maestro/ReleasePipelineRunner/ReleasePipelineRunner.cs

```diff
@@ -102,7 +102,7 @@ public async Task<TimeSpan> RunAsync(CancellationToken cancellationToken)
                 Logger.LogError(ex, "Processing queue messages");
             }
 
-            return TimeSpan.FromSeconds(1);
+            return TimeSpan.FromMinutes(1)
```

Comment:
```suggestion
            return TimeSpan.FromMinutes(1);
```

Reply:
Having the service wait for a minute between spurts of actually doing work seems like a really long time of doing nothing, but given how fragile the pipelines have proven to be, and that we routinely have to edit them manually to unblock things, I'm ok with this workaround until we know what the new version of this service will do once we move off the UI-Based release pipelines.

Reply:
Thanks for the feedback :-)